kamon-play-2.5.x/src/main/scala/kamon.play/instrumentation/RequestInstrumentation.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * =========================================================================================
+ * Copyright © 2013-2017 the kamon project <http://kamon.io/>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ * =========================================================================================
+ */
+
+package kamon.play.instrumentation
+
+import kamon.agent.libs.net.bytebuddy.description.method.MethodDescription
+import kamon.agent.libs.net.bytebuddy.matcher.ElementMatcher.Junction
+import kamon.agent.libs.net.bytebuddy.matcher.ElementMatchers._
+import kamon.agent.scala.KamonInstrumentation
+import kamon.play.instrumentation.advisor.{FiltersFieldAdvisor, RouteRequestAdvisor}
+import kamon.play.instrumentation.interceptor.{ErrorInterceptor, GlocalSettingsFiltersInterceptor}
+import kamon.play.instrumentation.mixin.InjectTraceContext
+import play.api.mvc.EssentialFilter
+
+class RequestInstrumentation extends KamonInstrumentation {
+
+  val DefaultHttpRequestHandlerConstructor: Junction[MethodDescription] = isConstructor()
+    .and(takesArgument(3, classOf[Seq[EssentialFilter]]))
+  val RouteRequestMethod: Junction[MethodDescription] = named("routeRequest")
+  val FiltersMethod: Junction[MethodDescription] = named("filters")
+  val OnServerOrOnClientErrorMethod: Junction[MethodDescription] = named("onClientError").or(named("onServerError"))
+
+  forSubtypeOf("play.api.mvc.RequestHeader") { builder =>
```

Comment:
Also here we can improve our api, all our instrumentations are similar and we can hide some verbosity making some methods in `KamonInstrumentation` for example: 
```scala
val RouteRequestMethod: Junction[MethodDescription] = named("routeRequest")

should be:

val RouteRequestMethod = methodNamed("routeRequest") 

```
or something like this and in this way we can hide all the imports of `import kamon.agent.libs.net.bytebuddy`